clang/tools/clang-format/git-clang-format

```diff
@@ -869,5 +875,12 @@ def convert_string(bytes_input):
         return str(bytes_input)
 
 
+def print_filename(filename, null=False):
+    if null:
+        print(filename + "\0", end="")
```

Comment:
Shouldn't you copy the the indentation of the printed files?

Reply:
I don't think so, because indentation is meaningless in binary output. You want to feed this to a program like `xargs -0`. This is useful when you want to pipe the binary data of a list of filepaths as-is to a tool. Adding tab characters in there would only confuse and make it necessary to crop the tab characters before piping it.

Reply:
> Shouldn't you copy the the indentation of the printed files?

It prints the filename, not the file.